src/lib/chat/matrix-client.ts

```diff
@@ -313,6 +314,28 @@ export class MatrixClient implements IChatClient {
     return await this.getMessageByRoomId(roomId, editResult.event_id);
   }
 
+  async markRoomAsRead(roomId: string): Promise<void> {
+    const room = this.matrix.getRoom(roomId);
+
+    if (!room) {
+      return;
+    }
+
+    const events = room.getLiveTimeline().getEvents();
+    const latestEvent = events[events.length - 1];
+
+    if (!latestEvent) {
+      return;
+    }
+
+    try {
+      await this.matrix.sendReadReceipt(latestEvent);
+      await this.matrix.setRoomReadMarkers(roomId, latestEvent.event.event_id);
+    } catch (error) {
+      console.error(`Failed to mark room with ID: ${roomId} as read. Error:`, error);
```

Comment:
just wondering, does this go to sentry right now? since we're doing most of this stuff from the frontend now (instead of the zero-api). cc: @dalefukami